c++/src/kj/async-inl.h

```diff
@@ -1434,14 +1480,17 @@ inline Promise<T> constPromise() {
   return _::PromiseNode::to<Promise<T>>(_::OwnPromiseNode(&NODE));
 }
 
+Promise<void> yield();
+Promise<void> yieldHarder();
```

Comment:
Per my other comment I don't think this is the right name for a public API. Proposed `yieldUntilQueueEmpty()` as an alternative.

Also, these declarations should go in async.h and have doc comments, now that they are public.

Reply:
Sorry, I forgot about the renaming. Fixed that, moved the functions and added some brief doc comments (open to suggestions on those).
Turns out there already was a TODO to make yield() public, resolved that too.